nixos/modules/services/networking/unifi.nix

```diff
@@ -21,16 +23,16 @@ in
     services.unifi.enable = mkOption {
       type = types.bool;
       default = false;
-      description = lib.mdDoc ''
+      description = mdDoc ''
```

Comment:
It's actually the opposite that is recommended, see #208242.

Reply:
That was a funny plot twist. I've now retconned the first commit (now 2bc02c0) to push in the right direction instead.